packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/Update/UpdateTooltip.tsx

```diff
@@ -0,0 +1,168 @@
+import styled, { useTheme } from 'styled-components';
+import { useDevice, useSelector } from '../../../../../../../hooks/suite';
+import { Column, Row, Icon, iconSizes, Text, getColorForIconVariant } from '@trezor/components';
+import { ReactNode } from 'react';
+import {
+    mapTrezorModelToIcon,
+    mapUpdateStatusToVariant,
+    UpdateStatus,
+    UpdateVariant,
+    mapUpdateStatusToIcon,
+} from './updateQuickActionTypes';
+import { Translation } from '../../../../../Translation';
+import { borders, spacings, spacingsPx } from '@trezor/theme';
+import { TranslationKey } from '@suite-common/intl-types';
+import { getFirmwareVersion } from '@trezor/device-utils';
+import { isDesktop } from '@trezor/env-utils';
+
+type IconCircleWrapperProps = {
+    $variant: UpdateVariant;
+};
+
+const IconCircleWrapper = styled.div<IconCircleWrapperProps>`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: 14px;
+    height: 14px;
+
+    background-color: ${({ theme, $variant }) =>
+        getColorForIconVariant({ theme, variant: $variant })};
+    border-radius: ${borders.radii.full};
+    border: 1px solid ${({ theme }) => theme['borderElevationNegative']};
+    padding: ${spacingsPx.xxxs};
+`;
+
+type UpdateRowProps = {
+    children: ReactNode;
+    leftIcon: ReactNode;
```

Comment:
How about using type `IconName` like everywhere else in case of icons? And if you want to add different component you could rename it to `leftItemComponent`. WDYT?